### PR TITLE
chore(helm): move to dedicated helm-repo

### PR DESCRIPTION
**Pull request by MichaelSp**
Moving to https://github.com/casdoor/casdoor-helm as suggested here: https://github.com/casbin/casdoor/pull/2583#issuecomment-1880058930

Goes along with the import: https://github.com/casdoor/casdoor-helm/pull/1

### CODE REVIEW

**Comment by hsluoyz** [comment]
@MichaelSp how this repo interacts with https://github.com/casdoor/casdoor-helm ?

@sp71

**Comment by MichaelSp** [comment]
@hsluoyz via the build workflow: https://github.com/casbin/casdoor/pull/2587/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R209-R230

**Comment by hsluoyz** [comment]
@MichaelSp where is the secret key for `git push origin HEAD:master`?

**Comment by sp71** [comment]
Thoughts on instead triggering a Github [Dispatch](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch) or [repo-dispatch](https://github.com/peter-evans/repository-dispatch) instead? Then the helm repo can handle the event and update accordingly.

**Comment by MichaelSp** [comment]
Sorry, forgot to mention that: You can do `push` or `dispatch`,  you still need to provide a token from your [bot](https://github.com/casbin-bot) because both operations need to be explicitly authorized.

The `dispatch` option might be a bit more flexible, because you can also manually trigger if you need to.

**Comment by hsluoyz** [comment]
@sp71 what's the difference?

@MichaelSp I didn't see where the token is

**Comment by MichaelSp** [comment]
@hsluoyz I made it a bit more explicit: https://github.com/casbin/casdoor/pull/2587/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R215 the token would be named `ACTIONS_GITHUB_TOKEN`

**Comment by hsluoyz** [comment]
@MichaelSp use existing names instead of `ACTIONS_GITHUB_TOKEN`. See this repo or other repos

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -208,3 +206,28 @@ jobs:
           platforms: linux/amd64
           push: true
           tags: casbin/casdoor-all-in-one:${{steps.get-current-tag.outputs.tag }},casbin/casdoor-all-in-one:latest
+
+      - uses: actions/checkout@v3
+        if: steps.should_push.outputs.push=='true'
+        with:
+          repository: casdoor/casdoor-helm
+          ref: 'master'
+          token: ${{ secrets.ACTIONS_GITHUB_TOKEN }}
```
Comment (MichaelSp) [inline .github/workflows/build.yml:215]:
```suggestion
          token: ${{ secrets.GH_BOT_TOKEN }}
```

**Comment by hsluoyz** [comment]
@MichaelSp does this docs: https://casdoor.org/docs/basic/try-with-helm need to update?

**Comment by MichaelSp** [comment]
@hsluoyz kind of, which is rather unrelated to the changes here. Some improvement suggestions:
 
 * link the `values.yaml` to https://github.com/casdoor/casdoor-helm/blob/master/charts/casdoor/values.yaml
 * link to https://hub.docker.com/r/casbin/casdoor-helm-charts/tags to let people figure out the latest version
 * the proposed `helm repo add casdoor https://hub.docker.com/repository/docker/casbin/casdoor-helm-charts` did not work for me. Use commands from https://hub.docker.com/layers/casbin/casdoor-helm-charts/0.4.0/images/sha256-de878a2843964f2913e32b8d28a0e9ef8728e16da309ebb987389047cd32f0dd?context=explore instead 

BTW: I'm using it with flux like this:
```yaml
apiVersion: source.toolkit.fluxcd.io/v1beta2
kind: HelmRepository
metadata:
  name: casbin
  namespace: auth
spec:
  interval: 48h
  url: oci://registry-1.docker.io/casbin
  type: "oci"
```

**Comment by hsluoyz** [comment]
@MichaelSp sounds good. Can you make a PR to: https://github.com/casdoor/casdoor-website to update these?

**Pull request merged**